packages/css/src/components/Form/form.css

```diff
@@ -300,3 +301,8 @@ textarea.form-input + .form-icon {
 .input-group-adornment + .form-field {
   border-radius: 0 var(--form-input-border-radius) var(--form-input-border-radius) 0;
 }
+
+/* SELECT */
+.form-field.disabled .select-icon {
+  color: var(--form-disabled-icon-color);
+}
```

Comment:
I think this should apply to all icons, not just the select

Right now disabling the Error field uses `--form-icon-color`  
![image](https://user-images.githubusercontent.com/12156624/222114059-ec2079ae-d6b2-4e79-8b34-477fabeab37b.png) 
![image](https://user-images.githubusercontent.com/12156624/222114106-17b8ed5a-6e1b-4442-921d-94edc3a969f4.png)

```suggestion
.form-field.disabled .form-icon {
  color: var(--form-disabled-icon-color);
}
```

Though this should probably be moved up to be next to the rest of the `.form-field` or `.form-icon` definitions

Reply:
Agree!